src/library_wasmfs.js

```diff
@@ -349,8 +371,20 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       __wasmfs_readdir_finish(state);
       return entries;
     }),
-    // TODO: mount
-    // TODO: unmount
+    mount: (type, opts, mountpoint) => {
+#if ASSERTIONS
+      if (typeof type == 'string') {
+        // The filesystem was not included, and instead we have an error
+        // message stored in the variable.
+        throw type;
+      }
+#endif
+      var backendPointer = type.createBackend(opts);
+      return FS.handleError(withStackSave(() => __wasmfs_mount(stringToUTF8OnStack(mountpoint), backendPointer)));
+    },
+    unmount: (mountpoint) => (
+      FS.handleError(withStackSave(() => __wasmfs_unmount(stringToUTF8OnStack(mountpoint))))
```

Comment:
```suggestion
      FS.handleError(withStackSave(() => __wasmfs_unmount(stringToUTF8OnStack(mountpoint))));
```

Reply:
I think adding a `;` here gives a compilation error. IIRC using `(mountpoint) => ( ... )` in JS expects only one line and will give errors if it sees `;`, while using `=> { ... }` is for multi-line arrow functions where `;` should be used after each line. Could that be the issue? If so, would it be better to use `() => { ... }` everywhere for consistency?

Reply:
Oh, I didn't see this was inside `( )` and not `{ }`. It's ok as it is I think.